### PR TITLE
Update documentation for languages

### PR DESCRIPTION
**Pull request by matbmapspeople**
I updated languages section for documentation with some text and screenshots.

### CODE REVIEW

**Comment by matbmapspeople** [comment]
@ammapspeople and @andreeaceachir142  it's ready for review

File: sdks-and-frameworks/web/map-management/display-language.md [hunk 1]
```diff
@@ -1,16 +1,36 @@
-# Language
+# Languages
 
-The language of MapsIndoors is independent of the chosen language on the device on which the app is used. This means that you need to explicitly tell MapsIndoors which language to use.
+MapsIndoors languages are independent of the chosen languages on the device on which the app is used. This means that you need to explicitly tell MapsIndoors which language to use.
+Example blow shows **'Languages'** section and dropdown from which you can select languages:
```
Comment (andreeaceachir142) [inline sdks-and-frameworks/web/map-management/display-language.md:4]: I guess you mean below 😁

**Comment by ammapspeople** [comment]
@matbmapspeople Please resolve the merge conflicts first.

File: sdks-and-frameworks/web/map-management/display-language.md [hunk 2]
```diff
@@ -1,16 +1,36 @@
...
+<div>
+    <img src="../../../.gitbook/assets/languages-default-view.png" alt="">
+</div>
+When solution has only one language, this language will be the default one. You are able to add as many languages as you want to and choose which one should be the default one.
```
Comment (andreeaceachir142) [inline sdks-and-frameworks/web/map-management/display-language.md:8]:
```suggestion
When a solution has only one language, that language will be the default one. You are able to add as many languages as you want to and choose which one should be the default one.
```

File: sdks-and-frameworks/web/other-guides/display-language-1.md [hunk 3]
```diff
@@ -1,16 +1,36 @@
-# Language
+# Languages
 
-The language of MapsIndoors is independent of the chosen language on the device on which the app is used. This means that you need to explicitly tell MapsIndoors which language to use.
+MapsIndoors languages are independent of the chosen languages on the device on which the app is used. This means that you need to explicitly tell MapsIndoors which language to use.
+Example blow shows **'Languages'** section and dropdown from which you can select languages:
```
Comment (ammapspeople) [inline sdks-and-frameworks/web/other-guides/display-language-1.md:4]:
```suggestion
The example below shows the **'Languages'** section and drop-down from which you can select languages:
```

File: sdks-and-frameworks/web/other-guides/display-language-1.md [hunk 4]
```diff
@@ -1,16 +1,36 @@
...
+<div>
+    <img src="../../../.gitbook/assets/languages-default-view.png" alt="">
+</div>
+When solution has only one language, this language will be the default one. You are able to add as many languages as you want to and choose which one should be the default one.
```
Comment (ammapspeople) [inline sdks-and-frameworks/web/other-guides/display-language-1.md:8]:
```suggestion
When a solution has only one language, this language will be the default one. You are able to add as many languages as you want to and choose which one should be the default one.
```
or maybe `your solution`?

File: sdks-and-frameworks/web/other-guides/display-language-1.md [hunk 5]
```diff
@@ -1,16 +1,36 @@
...
 
-Once your language of choice has been created, you can add the translation by clicking on any POI, which will open a menu on the left side of the screen. Here, you will see the following menu point, where you can enter translations for the languages you wish. If a field is left empty, the fallback language is English. In the example below, English (en) and Danish (da) are the enabled languages. (<mark style="background-color:red;">Hvilket example?</mark>)
+Once language of choice has been added as supported language, you can add the translation by clicking on any POI, which will open a menu on the left side of the screen. Here, you will see the following menu point, where you can enter translations for the languages you wish. If a field is left empty, the fallback language is the default one. In the example below, English (en) and Danish (da) are the enabled languages:
```
Comment (ammapspeople) [inline sdks-and-frameworks/web/other-guides/display-language-1.md:29]:
```suggestion
Once your language of choice has been added as a supported language, you can add the translation by clicking on any POI, which will open a menu on the left side of the screen. Here you will see the following menu point, where you can enter translations for the languages you wish. If a field is left empty, the fallback language is the default one. In the example below, English (en) and Danish (da) are the enabled languages:
```

**Comment by matbmapspeople** [comment]
All suggestions applied @ammapspeople and @andreeaceachir142

File: sdks-and-frameworks/web/other-guides/display-language-1.md [hunk 6]
```diff
@@ -1,16 +1,36 @@
...
+The example below shows the **'Languages'** section and drop-down from which you can select languages:
+<div>
+    <img src="../../../.gitbook/assets/languages-default-view.png" alt="">
+</div>
+When a solution has only one language, that language will be the default one. You are able to add as many languages as you want to and choose which one should be the default one.
+Adding German and Danish languages:
 
-If you do not specify a language, MapsIndoors will show information in the default language defined in the MapsIndoors CMS <mark style="background-color:red;">(Giver det mening at fortælle hvor i CMSet?)</mark>. Likewise, if you specify a language that is not supported, MapsIndoors will also show information in the default language. <mark style="background-color:red;">Skal vi nævne hvilke languages, der rent faktisk er supported? Eller er det nok, at det står nedenunder.</mark>
+<div>
+<img src="../../../.gitbook/assets/languages-adding-languages.png" alt="">
```
Comment (ammapspeople) [inline sdks-and-frameworks/web/other-guides/display-language-1.md:12]:
```suggestion
    <img src="../../../.gitbook/assets/languages-adding-languages.png" alt="">
```

File: sdks-and-frameworks/web/other-guides/display-language-1.md [hunk 7]
```diff
@@ -1,16 +1,36 @@
...
+</div>
+
+Setting German as default language:
+
+<div>
+<img src="../../../.gitbook/assets/languages-setting-default-language.png" alt="">
```
Comment (ammapspeople) [inline sdks-and-frameworks/web/other-guides/display-language-1.md:18]:
```suggestion
    <img src="../../../.gitbook/assets/languages-setting-default-language.png" alt="">
```

File: sdks-and-frameworks/web/other-guides/display-language-1.md [hunk 8]
```diff
@@ -1,16 +1,36 @@
...
 
-Once your language of choice has been created, you can add the translation by clicking on any POI, which will open a menu on the left side of the screen. Here, you will see the following menu point, where you can enter translations for the languages you wish. If a field is left empty, the fallback language is English. In the example below, English (en) and Danish (da) are the enabled languages. (<mark style="background-color:red;">Hvilket example?</mark>)
+Once your language of choice has been added as a supported language, you can add the translation by clicking on any POI, which will open a menu on the left side of the screen. Here, you will see the following menu point, where you can enter translations for the languages you wish. If a field is left empty, the fallback language is the default one. In the example below, English (en) and Danish (da) are the enabled languages:
+
+<div>
+<img src="../../../.gitbook/assets/languages-english-danish-enabled.png" alt="">
```
Comment (ammapspeople) [inline sdks-and-frameworks/web/other-guides/display-language-1.md:32]:
```suggestion
    <img src="../../../.gitbook/assets/languages-english-danish-enabled.png" alt="">
```

**Review by ammapspeople — APPROVED** [review]

**Pull request merged**